L1Trigger/TrackFindingTracklet/test/BuildFile.xml

```diff
@@ -7,9 +7,15 @@
     <use   name="root"/>
     <use   name="heppdt"/>
     <use   name="CommonTools/UtilAlgos"/>
+    <use   name="CondFormats/Alignment"/>
```

Comment:
this seems to be a fail in merging. this build file has been cleaned up by central CMSSW so these changes should be reverted.